### DISCUSSION TITLE
useWatchAsset

### DISCUSSION THREAD

**Discussion by pociej** [wevm/wagmi#3103] [Ideas]
As @tmm  mentioned here 
https://github.com/wagmi-dev/wagmi/issues/969 

"... we should add a hook for this — something like useWatchAsset — so it's easy for others to use." 
I have something ready so I could make PR.

**Comment by tmm** [comment]
You are welcome to PR it into the next version of Wagmi at the [`alpha`](https://github.com/wagmi-dev/wagmi/tree/alpha) branch.
